### PR TITLE
Class variables more accessible

### PR DESCRIPTION
**Pull request by AustinEast**
Made it easier to edit both the speed and margin of the zoom on creation of the object, and added functions to change them in real time.

### CODE REVIEW

File: flixel/addons/display/FlxZoomCamera.hx [hunk 1]
```diff
@@ -29,7 +29,7 @@ class FlxZoomCamera extends FlxCamera
 	 */
 	private var _zoomMargin:Float;
 	
-	public function new(X:Int, Y:Int, Width:Int, Height:Int, Zoom:Float = 0)
+	public function new(X:Int, Y:Int, Width:Int, Height:Int, Zoom:Float = 0, ZoomSpeed:Float = 25, ZoomMargin:Float = 0.25)
```
Comment (Gama11) [inline flixel/addons/display/FlxZoomCamera.hx]: I feel like these don't necessarily need to be in the constructor, having a lot of arguments always has an impact on readability.

Plus, your forgot to actually use the parameter values to initialize the variables. ;)

File: flixel/addons/display/FlxZoomCamera.hx [hunk 2]
```diff
@@ -114,4 +114,18 @@ class FlxZoomCamera extends FlxCamera
 		flashSprite.scaleX = X;
 		flashSprite.scaleY = Y;
 	}
-}
\ No newline at end of file
+	/**
+	 * Given a value passed in, updates the speed of zooming. Useful for real time changes.
+	 */
+	public function setZoomSpeed(Value:Float):Void
```
Comment (Gama11) [inline flixel/addons/display/FlxZoomCamera.hx]: The variables should just be public instead of adding setter functions.

**Comment by Gama11** [comment]
You're targetting the wrong branch, any changes go to dev.

File: flixel/addons/display/FlxZoomCamera.hx (hunk 1 above)
Comment (AustinEast) [inline flixel/addons/display/FlxZoomCamera.hx]: Oops! I was copying from my local file and forgot to add that change in. I do feel at least the zoom speed is pretty useful to have there.

File: flixel/addons/display/FlxZoomCamera.hx (hunk 2 above)
Comment (AustinEast) [inline flixel/addons/display/FlxZoomCamera.hx]: That would work too.

Comment (Gama11) [inline flixel/addons/display/FlxZoomCamera.hx]: Done in 44499cb.

**Pull request closed without merging**